Design-Docs/glossary.md

```diff
@@ -0,0 +1,143 @@
+
+If you offer hyper-converged infrastructure, a compute host also serves the Ceph. This makes the storage dynamically scalable (horizontally and vertically). For the Ceph services, 1 CPU core and 4 GB of RAM are reserved per OSD to ensure appropriate performance. 
+
+## Definition of Manager Node
+
+From here, the OSISM Ansible playbooks are applied to the environment. Furthermore, the following services often run here non-redundantly: Prometheus server,....
+
+## Definition of provider network
+
+The provider network is the network that is "in front", i.e. at the output points of the openstack/SCS. This is usually a public network, but can also be a private network in individual cases. IPs from the provider network can be assigned to instances within the SCS. The same applies to load balancers, of course. 
+
+## Definition of API
+
+The Rest API provides the core of openstack/SCS  and can be addressed for a whole region. It accepts and responds to end-user API calls.  The service supports the OpenStack Compute API, the Amazon EC2 API, and a special Admin API for privileged users to perform administrative actions. Policies are enforced and most orchestration actions can be started, such as launching an instance.
+
+## Horizon
+
+Horizon is openstack's preferred GUI for the end user, but also for the administrator for a quick overview. It runs on the controller node. 
+
+## Message Queue
+
+Most OpenStack services communicate with each other through the message queue. For example, Compute communicates with Block Storage services and Network services via the message queue. RabbitMQ, Qpid, and Zeromq are popular choices for a message queue service. When the message queue fails or becomes inaccessible, the cluster generally comes to a halt and ends up in a read-only state where the information is stuck at the point where the last message was sent. Therefore, this is clustered.  RabbitMQ has shown itself to be the most widespread and best supported variant in the OpenStack context, Qpid occurs occasionally, ZeroMQ lacks HA functionality to date
+
+## Keystone
+
+The OpenStack Identity module called Keystone is used as an authentication and rights system between the OpenStack components. Keystone divides access to projects in the cloud into so-called "clients" ("tenant"). A tenant is a tenant of the cloud and has at least one associated user. It is possible to create multiple users per tenant with different rights. Keystone uses a token system for authorization and also supports connection to other authentication options such as LDAP. Furthermore, it is possible to abstract the authentication to the web server via an upstream web server and thus, for example, directly use one of the numerous existing authentication modules of the Apache web server for authorization.
+
```

Comment:
We should probably not have the word tenant in here.

Reply:
I think it might be good to link with the official keystone doc, which (IIRC) explains the external and other mechanisms for auth for keystone.

Reply:
Do we really need to re-define those upstream project's definitions?
Shouldn't those be referring to upstream documentations instead?